src/schema/v2/me/index.ts

```diff
@@ -210,8 +211,22 @@ export const meType = new GraphQLObjectType<any, ResolverContext>({
       resolve: (_root, _options, { meCreditCardsLoader }) => {
         if (!meCreditCardsLoader) return null
         return meCreditCardsLoader({ qualified_for_bidding: true }).then(
-          ({ body }) => {
-            return body && body.length > 0
+          ({ body: cards = [] }) => {
+            const unexpiredCards = cards.filter((card) => {
+              const {
+                expiration_month: expMonth,
+                expiration_year: expYear,
+              } = card
+
+              // Moment months are 0-indexed
+              const expirationMoment = moment({
+                year: expYear,
+                month: expMonth - 1,
+              }).endOf("month")
```

Comment:
#non-blocking. I think `moment(...)` will be local timezone. This exact moment may be a few hours different depending on the timezone. MP servers might be configured in the UTC timezone so it's still UTC, but potentially we can explicitly set the end of month moment to be UTC. Maybe something like
```
moment.utc(...)
```

Reply:
good catch!